### ISSUE TITLE
QuestHelper / LevelingGuide missing

### ISSUE THREAD

**Issue by madpew** [Open-KO/KnightOnline#97]
It's just not implemented, the Files are there (UI: ka_levelguide_us, Datatable: help_us.tbl).
There's no interaction with the server. This is a client-side feature.

- [ ] Load and show the UI from the "Q" Button (defined in UIStateBar)
- [ ] on search-press, filter the datatable according to the level and update the UI with the info
- [ ] make paging work

Should the Q-Button be hidden if the client can't load the UI or the help-table?

**Comment by twostars** [comment]
>> Should the Q-Button be hidden if the client can't load the UI or the help-table?
No, because that case is exceptional. Those cases won't (or shouldn't) happen when it's implemented and such behaviour is implemented. No real reason to (temporarily?) hide it.

**Comment by twostars** [comment]
CUILevelGuide was implemented by #256
